python/windy_gridworld.py

```diff
@@ -376,6 +408,11 @@ def save_value_and_policy_function_plot(self, algorithm):
                 marker="o",
             )
 
+        if algorithm == POLICY_ITERATION:
+            # TODO Temporary, just trying to visualize the locations where there are ties
+            locations_with_ties_x, locations_with_ties_y = list(zip(*self.locations_with_ties))
+            plt.plot(locations_with_ties_x, locations_with_ties_y, color="blue", marker="o", linestyle="None")
```

Comment:
Instead of blue points, add the optimal actions to the quiver plots (multiple arrows showing the optimal directions)